src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/StateFlowEngine.java

```diff
@@ -398,7 +238,7 @@ public StateFlow build(FlowState.Main initialState) {
                        flags.put(FlowFlag.RESPONDENT_RESPONSE_LANGUAGE_IS_BILINGUAL.name(), true)
                    )
                 .transitionTo(FULL_DEFENCE).onlyIf(fullDefenceSpec.and(not(contactDetailsChange)).and(not(isRespondentResponseLangIsBilingual))
-                                                       .and(not(pastClaimNotificationDeadline)))
+                                                       .and(not(pastClaimNotificationDeadline)).and(not(judgeOrderVerificationRequired)))
```

Comment:
Flag is added only in one state. What will happen for example if the state transit "IN_MEDIATION" to  "CASE_DISCONTINUED"?